hyperon_das/utils.py

```diff
@@ -101,3 +102,9 @@ class QueryAnswer:
 def get_package_version(package_name: str) -> str:
     package_module = import_module(package_name)
     return getattr(package_module, '__version__', None)
+
+def serializer(payload: Any) -> bytes:
```

Comment:
These methods should be named `serialize()` and `deserialize()`. Which is a standard (and makes more sense in English)